.gitignore

```diff
@@ -63,3 +63,7 @@
 
 /tags
 /src/TAGS
+src/TAGS
```

Comment:
I just find it strange that all paths above start with `/` and paths below without `/`, and we have both `tags` and `tags`.